galley/pkg/config/analysis/local/analyze.go

```diff
@@ -102,6 +106,7 @@ func NewSourceAnalyzer(m *schema.Metadata, analyzer *analysis.CombinedAnalyzer,
 
 	sa := &SourceAnalyzer{
 		m:                    m,
+		meshCfg:              meshcfg.Default(),
```

Comment:
is this still using some random default mesh config in galley code that isn't used by the rest of istio (namely pilot)? I think we changed it but not certain

Reply:
The `meshcfg.Default()` being invoked here appears to only be used by Galley. 
https://github.com/istio/istio/blob/master/galley/pkg/config/meshcfg/defaults.go

It looks like the only difference between it and `mesh.DefaultMeshConfig` (which does appear to be used by Pilot as well) is that Galley's version has the ingress stuff turned on. 

If this is something we need to change, it should be handled outside the scope of this PR.